packages/components/pie-text-input/README.md

```diff
@@ -19,23 +19,23 @@
 7. [Slots](#slots)
 8. [Contributing](#contributing)
 
-## pie-input
+## pie-text-input
 
-`pie-input` is a Web Component built using the Lit library.
+`pie-text-input` is a Web Component built using the Lit library.
 
 This component can be easily integrated into various frontend frameworks and customized through a set of properties.
 
 
 ## Installation
 
-To install `pie-input` in your application, run the following on your command line:
+To install `pie-text-input` in your application, run the following on your command line:
 
 ```bash
 # npm
-$ npm i @justeattakeaway/pie-input
+$ npm i @justeattakeaway/pie-text-input
 
 # yarn
-$ yarn add @justeattakeaway/pie-input
+$ yarn add @justeattakeaway/pie-text-input
```

Comment:
Should we have a note  mentioning that versions older than the next one will be only available under the previous package name?

Reply:
Just pushed a change @fernandofranca, what do you think>